src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/UtilityTypeExtensions.cs

```diff
@@ -44,20 +45,32 @@ private static IEnumerable<AggregateType> TypeAndBaseClassInterfaces(this Aggreg
                     yield return t;
         }
 
-        public static IEnumerable<CType> AllPossibleInterfaces(this CType type)
+        public static IEnumerable<AggregateType> AllPossibleInterfaces(this CType type)
         {
             Debug.Assert(type != null);
-            if (type.IsAggregateType())
+            if (type is AggregateType ats)
```

Comment:
It isn't, but it will be soon. I think the (unnecessary) `partial` on it caused it to miss the find-replace I used to seal a lot of them in January.